### PR TITLE
Support for adding scoped CSS without triggering global refresh

### PR DESCRIPTION
**Pull request by joshcomley**
As per the problem I reported here:
NativeScript/theme#179

When using component scoped CSS there is no need to trigger a full global refresh of the styles.

This change does not affect performance or function of using `encapsulation: ViewEncapsulation.None` on a component.

This PR goes hand in hand [with a modification to `tns-core-modules`](https://github.com/NativeScript/NativeScript/pull/7907) to support the below changes.

## PR Checklist

- [ ] The PR title follows our guidelines: https://github.com/NativeScript/NativeScript/blob/master/CONTRIBUTING.md#commit-messages.
- [ ] There is an issue for the bug/feature this PR is for. To avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it.
- [ ] You have signed the [CLA](http://www.nativescript.org/cla).
- [ ] All existing tests are passing: https://github.com/NativeScript/nativescript-angular/blob/master/DevelopmentWorkflow.md#running-the-tests
- [ ] Tests for the changes are included.

## What is the new behavior?

Fixes/Implements/Closes #[Issue Number].

### CODE REVIEW

**Comment by joshcomley** [comment]
I am happy to resubmit this PR following the above guidelines (I am not entirely sure how to compose the PR title, for example), but would somebody be able to review the code to ensure it's a valid PR otherwise? Many thanks

File: nativescript-angular/renderer.ts [hunk 1]
```diff
@@ -365,6 +365,11 @@ const addStyleToCss = profile('"renderer".addStyleToCss', function addStyleToCss
     addCss(style);
 });
 
+// tslint:disable-next-line
+const addScopedStyleToCss = profile('"renderer".addStyleToCss', function addStyleToCss(style: string): void {
```
Comment (MartoYankov) [inline nativescript-angular/renderer.ts]:
```suggestion
const addScopedStyleToCss = profile('"renderer".addStyleToCss', function addScopedStyleToCss(style: string): void {
```

File: nativescript-angular/renderer.ts [hunk 2]
```diff
@@ -365,6 +365,11 @@ const addStyleToCss = profile('"renderer".addStyleToCss', function addStyleToCss
     addCss(style);
 });
 
+// tslint:disable-next-line
+const addScopedStyleToCss = profile('"renderer".addStyleToCss', function addScopedStyleToCss(style: string): void {
```
Comment (MartoYankov) [inline nativescript-angular/renderer.ts]:
```suggestion
const addScopedStyleToCss = profile('"renderer".addScopedStyleToCss', function addScopedStyleToCss(style: string): void {
```

Sorry I didn't notice this before.

**Comment by MartoYankov** [comment]
@VladimirAmiorkov  This PR depends on https://github.com/NativeScript/NativeScript/pull/7907 , which will be released with {N} 6.2.

**Comment by VladimirAmiorkov** [comment]
test tns_core_modules#next

**Comment by VladimirAmiorkov** [comment]
All checked have passed. This PR is ready for merge once tns-core-modules@6.2.0 is released.

**Comment by joshcomley** [comment]
@VladimirAmiorkov thanks - does that mean this will make it into 6.2.0 or it will be in a subsequent release?

**Comment by VladimirAmiorkov** [comment]
@joshcomley Yes this will make it in time for NativeScript 6.2.0.

**Comment by joshcomley** [comment]
@VladimirAmiorkov is there an estimated date for the 6.2.0 release? I understand this isn't exactly the thread to ask, but I'm not sure where is!

**Comment by MartoYankov** [comment]
@joshcomley We are aiming for the end of the month.

**Comment by elena-p** [comment]
test tns_core_modules#next

**Pull request merged**